apps/mobile/src/components/screen/screen.tsx

```diff
@@ -21,12 +21,14 @@ function Body(props: BoxProps) {
 }
 
 function Title({ children }: HasChildren) {
-  return (
+  return typeof children === 'string' ? (
```

Comment:
Modifying this slightly so the `TokenOverview` icon can trigger the header animation

https://github.com/user-attachments/assets/65a97cda-dee3-4d88-9237-8e389ca0e378